woocommerce/woocommerce-android
# [Mobile Payments] Payments hub to the right package and appearance animation fix

### Description

The PR:
* Renames `CardReaderHub*` -> `PaymentsHub*`
* Moves `PaymentsHub*` from `cardreader` package to `payments` package
* Makes `Reviews` entrance animation match the other animations
* Fixes animation of the `PaymentsHub` appearance

Easier to review commit by commit as most of the changes just moving things around

### Testing instructions

* More -> Payments -> notice the fixed animation
* More -> Reviews -> notice the animation that matches other animations on the screen

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/4923871/c709ba13-894d-4082-8964-f1f6b2a14041
 
https://github.com/woocommerce/woocommerce-android/assets/4923871/f8e389d4-6ac2-4ad5-9efc-b73c3958a593 

---

https://github.com/woocommerce/woocommerce-android/assets/4923871/cf9a1aef-d7e6-4bf2-9b3a-b301be2211dd

https://github.com/woocommerce/woocommerce-android/assets/4923871/1a445653-90c1-4877-ad8c-59c9d5187e26
